documentation/modules/cruise-control/con-optimization-goals.adoc

```diff
@@ -184,7 +188,9 @@ User-provided optimization goals must:
 * Include all configured xref:hard-soft-goals[hard goals], or an error occurs
 * Be a subset of the master optimization goals
 
-To ignore the configured hard goals in an optimization proposal, add the `skipHardGoalCheck: true` option to the `KafkaRebalance` custom resource.
+To ignore the configured hard goals in the generated optimization proposal, add the `skipHardGoalCheck: true` option to the `KafkaRebalance` custom resource.
+
+To apply an optimization proposal and rebalance the Kafka cluster, you annotate the `KafkaRebalance` custom resource. 
```

Comment:
should we be explaining the annotation a bit more here or linking somewhere?

Reply:
Thanks for flagging this up. I've removed this reference to annotating / approving the `KafkaRebalance` CR and added a cross-ref to the generating optimization proposals procedure. That procedure now includes an optional step for adding `skipHardGoalCheck: true` to the CR.